jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/src/test/resources/org/jbpm/test/container/archive/registerrestservice/WEB-INF/weblogic.xml

```diff
@@ -18,7 +18,7 @@
              <package-name>org.codehaus.jettison.*</package-name>
 
              <!-- jsr311*.jar -->
-             <package-name>javax.ws.rs.*</package-name>
+             <!--<package-name>javax.ws.rs.*</package-name>-->
```

Comment:
Could we remove it instead of commenting out

Reply:
Comment added.